ccd-definition/AuthorisationCaseField/court-admin.json

```diff
@@ -642,5 +642,12 @@
     "CaseFieldID": "evidenceHandled",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "scannedDocuments",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRUD"
```

Comment:
probably R should be enough for this role

Reply:
yep, knowing courtadmin will have bulkscan role I can update permissions